tasks.py

```diff
@@ -1214,6 +1217,7 @@ def __publish_version(ctx, func, description):
 
 
 def __build_function(ctx, func, upload_to_s3=False):
+    print(f"Building {func} function")
```

Comment:
Add print statements that explain the steps and hide=1 to stop the terminal spam